src/tenant/tests/test_initialization.py

```diff
@@ -141,6 +141,27 @@ def test_default_message_quest_notifies_owner(self):
         owner = User.objects.filter(username="owner", is_staff=True).first()
         self.assertEqual(message_quest.specific_teacher_to_notify, owner)
 
+    def test_default_tags__quest(self):
```

Comment:
our naming convention for test methods is:
`test_name_of_method__what_is_being_tested`

i.e. `test`+ underscore + `name_of_method_being_tested` + double underscore + `what_is_being_tested_in_the_method`

So for this I think it would be:

`test_create_orientation_campaign__default_tags_created()`

Reply:
done